pkg/gofr/datasource/clickhouse/clickhouse.go

```diff
@@ -165,30 +166,25 @@ type Health struct {
 	Details map[string]interface{} `json:"details,omitempty"`
 }
 
-func (h *Health) GetStatus() string {
-	return h.Status
-}
+const errStatusDown = "StatusDown"
```

Comment:
```suggestion
var errStatusDown = errors.New("status down")
```

Usually err is a prefix for errors, not string.

Also, please move this sentinel error to an higher level in the package, this way it can be shared by all sub packages

Reply:
We would be moving that at single place as part of a single PR, with major refactors.

Reply:
it would make sense, yes